.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,158 @@
+          https://github.com/keymanapp/keyman/issues?q=is%3Aopen+is%3Aissue+label%3Abug
+
+        * If you find a matching issue, go ahead and add a comment on that issue.
+        * If you find related issues, make sure you list them below.
+
+        You may also wish to discuss your bug report with the Keyman community at:
+          https://community.software.sil.org/c/keyman
+
+  - type: textarea
+    id: description
+    attributes:
+      label: "Describe the bug"
+      description: "A clear and concise description of the bug. Include details of affected applications if relevant."
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        When reporting on issues with unexpected output, it is very helpful to include the exact sequence of keys, the output expected, and the output received, for example:
+
+        ```
+        Typed: <kbd>a</kbd> <kbd>b</kbd> <kbd>Shift</kbd>+<kbd>C</kbd>
+        Received: "abc"
+        Expected: "abC"
+        ```
+
+        It's optional to use `<kbd></kbd>` to get key icons in the keying sequence, but it does add clarity; alternatively, do something like this:
+
+        ```
+        Typed: a b shift+C
+        ```
+
+        Tip: use key names such as `spacebar` rather than just a space!
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: "Reproduce the bug"
+      description: |
+        Write up the steps you took which triggered the bug. Be as precise as possible; you can paste screenshots and/or videos if you wish.
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: "Expected behavior"
+      description: "A clear and concise description of what you expected to happen."
+
+  - type: textarea
+    id: related-issues
+    attributes:
+      label: "Related issues"
+      description: "After completing your issue search, you may find some that seem related; list them here"
+
+  - type: markdown
+    attributes:
+      value: ---
+
+  - type: checkboxes
+    id: context-product
+    attributes:
+      label: "Keyman apps"
+      description: "Which Keyman app(s) are you experiencing this problem with?"
+      options:
+        - label: Keyman for Android
+        - label: Keyman for iPhone and iPad
+        - label: Keyman for Linux
+        - label: Keyman for macOS
+        - label: Keyman for Windows
+        - label: Keyman Core
+        - label: Keyman Developer
+        - label: Keyman Engine for Android
+        - label: Keyman Engine for iPhone and iPad
+        - label: Keyman Engine for Linux
+        - label: Keyman Engine for macOS
+        - label: Keyman Engine for Web
+        - label: Keyman Engine for Windows
```

Comment:
I'm not sure if having `Keyman Engine for *` as separate options is helpful or just confusing. It's not so obvious which one you should pick.

Reply:
Yep, agree, we can probably triage into appropriate component quickly anyway.